### PR TITLE
{vis}[foss/2017b] OpenCV v3.3.0, GTK+ v2.24.32, ATK v2.27.1, ...

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/ATK/ATK-2.27.1-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['673a953987b301ab1e24e7d11677b6e7ba3226411a168449ba946765b6d44297']
+
+builddependencies = [
+    ('GObject-Introspection', '1.53.5', '-Python-2.7.14'),
+]
```
Comment (verdurin) [inline easybuild/easyconfigs/a/ATK/ATK-2.27.1-foss-2017b.eb]: @boegel - perhaps one for the style checker in future is whether to have (as I prefer) a line in between `builddependencies` and `dependencies`. It's not consistent in this PR.

Comment (boegel) [reply easybuild/easyconfigs/a/ATK/ATK-2.27.1-foss-2017b.eb]: Yes, that would certainly help making things more consistent.

A bigger one is checking the order of easyconfig parameters though, see https://github.com/easybuilders/easybuild-framework/pull/2165

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 5 (5 easyconfigs in this PR)
node2444.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bb49df1aab12fc20d681887e6e49d99e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2441.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e9628794605a100802589be6f1e3102a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2035.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/86bf7f221e2e682946b6eb5dbba9dcdb for a full test report.

**Pull request merged**